dotCMS/src/main/java/com/dotmarketing/factories/EmailFactory.java

```diff
@@ -437,11 +435,13 @@ public static WebForm sendParameterizedEmail(Map<String,Object> parameters, Set<
 				}
 
 				// Saving email backup in a file
-				try {
+                OutputStream os = null;
+                BufferedOutputStream bos = null;
+                try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)

Reply:
done